builder/Makefile.release

```diff
@@ -0,0 +1,109 @@
+OUTDIR ?= dist
+OUT_ARCHIVE := $(OUT_DIR)/tracee.tar.gz
+OUT_CHECKSUMS := $(OUT_DIR)/checksums.txt
+RELEASE_FILES := LICENSE $(OUT_DIR)/tracee-ebpf $(OUT_DIR)/tracee-rules $(OUT_DIR)/rules $(OUT_DIR)/tracee.bpf.core.o
+PUSH_DOCKER_REPO ?= aquasec/tracee
+PUSH_DOCKER_TAG ?= $(RELEASE_TAG:v%=%)
+
+CMD_DOCKER ?= docker
+CMD_CHECKSUM ?= sha256sum
+CMD_GITHUB ?= gh
+CMD_TAR ?= tar
+CMD_CP ?= cp
+CMD_GIT ?= git
+CMD_RM ?= rm
+CMD_MAKE ?= make
+
+TARGET_CORE_IMAGE_NAME = build-alpine-tracee-core
+TARGET_BTFHUB_IMAGE_NAME = build-alpine-tracee-core-btfhub
+TARGET_NOCORE_IMAGE_NAME = build-alpine-tracee-nocore
+
+MAKE_CONTAINER = $(CMD_MAKE) -f builder/Makefile.tracee-container
+MAKE_BUILDER_CONTAINER_PREPARE = $(CMD_MAKE) -f builder/Makefile.tracee-make ubuntu-prepare ARG="all"
+MAKE_BUILDER_CONTAINER = $(CMD_MAKE) -f builder/Makefile.tracee-make ubuntu-make ARG="all"
+
+.check_%:
+#
+	@command -v $* >/dev/null
+	if [ $$? -ne 0 ]; then
+		echo "missing required tool $*"
+		exit 1
+	else
+		touch $@ # avoid target rebuilds due to inexistent file
+	fi
+
+.PHONY: .check_release_tag
+.ONESHELL:
+.check_release_tag:
+#
+	@if [ -z "$(RELEASE_TAG)" ]; then
+		echo "missing required variable RELEASE_TAG"
+		exit 1
+	fi
+
+#
+# release
+#
+release_notes := $(OUT_DIR)/release-notes.txt
+.PHONY: release
+release: \
+	| .check_release_tag \
+	.check_$(CMD_GIT) \
+	.check_$(CMD_DOCKER) \
+	.check_$(CMD_CHECKSUM) \
+	.check_$(CMD_GITHUB) \
+	.check_$(CMD_TAR) \
+	.check_$(CMD_CP) \
+#
+	-rm $(release_notes)
+	echo '## Changelog' > $(release_notes)
+	$(CMD_GIT) log --pretty=oneline --abbrev=commit --no-decorate --no-color tags/$(shell $(CMD_GIT) describe --tags --abbrev=0)..HEAD >> $(release_notes)
+	echo '' >> $(release_notes)
+#
```

Comment:
All makefiles now have a "help" target, that displays something like the comments you have in your makefile. Would you mind creating the help target with descriptions ?